sdcm/nemesis.py

```diff
@@ -4012,15 +4012,30 @@ def check_encryption_fact(sstable_util_instance, expected_bool_value):
             assert encryption_results
             assert all(encryption_result is expected_bool_value for encryption_result in encryption_results)
 
+        def run_write_scylla_bench_load(write_cmd):
+            # NOTE: 'scylla-bench' runs 'truncate' operation when 'validate-data' is used in addition
```

Comment:
it's that a issue/bug to fix on s-b ? 
or I don't understand why are we o.k. with it failing ?

Reply:
It is concurrency.
The full error from the scylla-bench looks like this:

```
2023/10/03 15:53:49 Error during truncate: seastar::rpc::remote_verb_error (filesystem error: stat failed: No such file or directory [/var/lib/scylla/data/keyspace1/tmp_encrypted_table-c8b6abb0620411eeb3bbffe61f372cec/me-3g9w_184y_38t8w2qwwiu866t6n7-big-Scylla.db])
```

The same on the DB level:

```
2023-10-03T15:53:49+00:00 longevity-10gb-3h-dev-db-node-245363be-1      !ERR | scylla[14449]:  [shard 0:sl:d] sstable - Error while linking SSTable: /var/lib/scylla/data/keyspace1/tmp_encrypted_table-c8b6abb0620411eeb3bbffe61f372cec/me-3g9w_184y_38t8w2qwwiu866t6n7-big-Scylla.db to /var/lib/scylla/data/keyspace1/tmp_encrypted_table-c8b6abb0620411eeb3bbffe61f372cec/snapshots/1696348429426-tmp_encrypted_table/me-3g9w_184y_38t8w2qwwiu866t6n7-big-Scylla.db: std::filesystem::__cxx11::filesystem_error (error system:2, filesystem error: stat failed: No such file or directory [/var/lib/scylla/data/keyspace1/tmp_encrypted_table-c8b6abb0620411eeb3bbffe61f372cec/me-3g9w_184y_38t8w2qwwiu866t6n7-big-Scylla.db])
```

Sstables concurrently get removed. It is expected behavior.

Reply:
Also, our goal is to check that `existing` sstables are `encrypted`. It gets checked by further assertions.

Reply:
this whole using of `--validata-data` on write load, isn't clear to me.
also letting s-b do truncate, sound a bit off to me... don't something I would expected from a stress tool like that.

but for now, I'll leave it as is